src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -3761,7 +3765,7 @@ theorem getLsbD_replicate {n w : Nat} {x : BitVec w} :
     by_cases hi : i < w * (n + 1)
     · simp only [hi, decide_true, Bool.true_and]
       by_cases hi' : i < w * n
-      · simp [hi', ih]
+      · simp [-getLsbD_eq_getElem, hi', ih]
```

Comment:
```suggestion
      · rw [ih]
        simp_all
```
(I see what's going on here, but having to remove simp lemmas from simp sets always worries me. The better fix might be to prove `getElem_replicate` *first*, and then derive this lemma from it, but not now.)

Reply:
Fixed.